server/games/game.py

```diff
@@ -795,6 +799,26 @@ def report_army_stats(self, stats_json):
         self._army_stats_list = json.loads(stats_json)["stats"]
         self._process_pending_army_stats()
 
+    def is_visible_to_player(self, player: Player) -> bool:
+        if player == self.host:
+            return True
+
+        mean, dev = player.ratings[self.rating_type]
+        displayed_rating = mean - 3 * dev
+        if (
+            self.enforce_rating_range
+            and displayed_rating not in self.displayed_rating_range
+        ):
+            return False
+
+        if self.host is None:
+            return False
+
+        if self.visibility == VisibilityState.FRIENDS:
```

Comment:
Maybe prefer `self.visibility is VisibilityState.FRIENDS`?

Reply:
Yea IDK, I think it's fine. There were some other enums that I ended up refactoring to classes with constants and then I had to change everything to `==`. It's more flexible since it can be overridden with `__eq__`.